docs/source-app/workflows/add_server/any_server.rst

```diff
@@ -31,7 +31,8 @@ Any server that listens on a port, can be enabled via a work. For example, here'
         def do_GET(self):
             self.send_response(HTTPStatus.OK)
             self.end_headers()
-            html = "<h1 style='color: blue'> Hello lit world </div>"
+            # Make sure to pass the data as bytes to the self.wfile.write call
```

Comment:
@tchaton - Is this okay? I thought adding a comment will be better, as it's more about this line/example. Please let me know if it's not what you meant :)